roqoqo-qasm/src/interface.rs

```diff
@@ -18,7 +18,15 @@ use roqoqo::Circuit;
 use roqoqo::RoqoqoBackendError;
 
 // Pragma operations that are ignored by backend and do not throw an error
-const ALLOWED_OPERATIONS: &[&str; 2] = &["PragmaSetNumberOfMeasurements", "InputSymbolic"];
+const ALLOWED_OPERATIONS: &[&str; 7] = &[
+    "PragmaSleep",
```

Comment:
Should these be allowed or should they raise an error?

Reply:
Allowed. In the sense that they should not stop the process nor add anything to the output. They are ignored.